### ISSUE TITLE
ManiDP loss is nan

### ISSUE THREAD

**Issue by whz945565** [huawei-noah/GAN-pruning#18]
No matter how I train, the recognition rate is always 10%, and the loss is always nan. I tried training from scratch and still got this result.why？

**Comment by yehuitang** [comment]
The pre-trained model is usually required in this method.
